crates/ere-dockerized/src/lib.rs

```diff
@@ -0,0 +1,503 @@
+//! # Ere Dockerized
```

Comment:
Re  https://github.com/eth-act/ere/pull/66#discussion_r2242925511 add some comments and an example.